src/test/java/fr/openent/competences/security/AccessIfMyStructureTest.java

```diff
@@ -0,0 +1,67 @@
+package fr.openent.competences.security;
+
+import fr.openent.competences.security.utils.WorkflowActions;
+import fr.wseduc.webutils.http.Binding;
+import io.netty.handler.codec.http.DefaultHttpHeaders;
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.core.http.impl.HeadersAdaptor;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.entcore.common.user.UserInfos;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@RunWith(VertxUnitRunner.class)
+public class AccessIfMyStructureTest {
+    AccessIfMyStructure access;
+    HttpServerRequest request;
+    Binding binding;
+    MultiMap map;
+    List<String> structures;
+    UserInfos user = new UserInfos();
```

Comment:
```suggestion
    UserInfos user;
```
Tu fais déjà l'instanciation dans le setup